src/reporter.cc

```diff
@@ -97,18 +97,9 @@ void SendMetadataRequest(std::vector<json::value>&& entries,
     throw (boost::system::system_error) {
 
   if (entries.size() == 1) {
-    // Add a copy of the single entry, except for the `views` field. This allows
-    // us to sent a request to the batch endpoint when we have a single request.
-    // We're making use of the fact that duplicate metadata is ignored.
```

Comment:
Did you mean to delete this sentence?

Reply:
I like `We're making use of the fact that sending two identical requests is idempotent.`. WDYT?

Reply:
Sounds good. Added.